doc/releases/changelog-dev.md

```diff
@@ -159,31 +154,107 @@
 * `qml.ctrl` can be used with the `qml.qjit` decorator.
   [(#4726)](https://github.com/PennyLaneAI/pennylane/pull/4726)
 
-<h3>Improvements 🛠</h3>
+<h4>Decompose circuits into the Clifford+T gateset 🧩</h4>
 
-* `qml.expval` with large `Hamiltonian` objects is now faster and has a significantly lower memory footprint (and constant with respect to the number of `Hamiltonian` terms) when the `Hamiltonian` is a `PauliSentence`. That is due to the introduction of a specialized `dot` method in the `PauliSentence` class which performs `PauliSentence`-`state` products.
-  [(#4839)](https://github.com/PennyLaneAI/pennylane/pull/4839)
+* `qml.transforms.decompositions.sk_decomposition` method implements the Solovay-Kitaev algorithm for
+  approximately decomposing any single-qubit operation to Clifford+T basis.
+  [(#4801)](https://github.com/PennyLaneAI/pennylane/pull/4801)
+
+* The `qml.clifford_t_decomposition()` transform provides an approximate breakdown of an input
```

Comment:
The second bullet point will eventually replace the first, we're just waiting on #4802 to be merged first :+1: